lua/telescope/config.lua

```diff
@@ -151,6 +151,26 @@ append(
   - "ascending"]]
 )
 
+append(
+  "tiebreak",
+  function(_, current_entry, existing_entry)
+    return #current_entry.ordinal < #existing_entry.ordinal
+  end,
+  [[
+  tiebreak can be set to a function which decides how to break
+  a tie when two entries have the same score.
+  A simple function which always returns false would keep the
+  entries in the order they are found, so existing_entry
+  before current_entry.
+  Vice versa always returning true would place the current_entry
+  before the existing_entry.
+
+  Signature: function(picker,current_entry,existing_entry) -> boolean
+
+  Default: function that breaks the tie based on the length of the entry's ordinal
```

Comment:
We dont have wrapping for documentation here. So you need to manually make sure that the line isnt longer than 78.
And i think this line is too long